04_numeronym/shahriys/main_test.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var bufnum bytes.Buffer
+	outnum = &bufnum
+
+	main()
+	expected := `[a11y K8s abc]`
+	actual := bufnum.String()
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()" + expected + " " + actual)
+	}
+
```

Comment:
Little nit, but you don't need the extra white space at the end of each function. Keep this in mind in future.